test/inc/TestDef.h

```diff
@@ -15,9 +15,14 @@ static const std::wstring c_testFileExtension_Packaged = L".reuniontestfile-pack
 static const std::wstring c_testFilePhaseEventName = L"ReunionTestFilePhaseEventName";
 
 static const std::wstring c_testStartupPhaseEventName = L"ReunionTestStartupPhaseEventName";
-
 static const std::wstring c_testInstanceRedirectedPhaseEventName = L"ReunionTestInstanceRedirectedPhaseEventName";
 
+inline const winrt::hstring c_rawNotificationPayload = L"<toast></toast>";
+inline IID c_comServerId = winrt::guid("ccd2ae3f-764f-4ae3-be45-9804761b28b2"); // Value from PushNotificationsTestAppPackage ComActivator in appxmanifest.
+inline IID c_fakeComServerId = winrt::guid("00000000-0000-0000-0000-000000000001");
+inline const int c_pushTestWait = 3000;
+inline const int c_pushTestChannelWait = c_pushTestWait + 300000; // Need to wait 300000ms for channel request + 3000ms for application overhead 
```

Comment:
Only used in APITest.cpp

CONSIDER: Moving to APITest.cpp

Reply:
Moving test wait times to APITest.cpp